src/decapodeacset.jl

```diff
@@ -663,3 +686,69 @@ Resolve function overloads based on types of src and tgt.
 resolve_overloads!(d::SummationDecapode) =
   resolve_overloads!(d, op1_res_rules_2D, op2_res_rules_2D)
 
+function replace_names!(d::SummationDecapode, op1_repls::Vector{Pair{Symbol, Any}}, op2_repls::Vector{Pair{Symbol, Symbol}})
+  for (orig,repl) in op1_repls
+    for i in collect(incident(d, orig, :op1))
+      d[i, :op1] = repl
+    end
+  end
+  for (orig,repl) in op2_repls
+    for i in collect(incident(d, orig, :op2))
+      d[i, :op2] = repl
+    end
+  end
+  d
+end
+
+ascii_to_unicode_op1 = Pair{Symbol, Any}[
+                        (:dt       => :∂ₜ),
```

Comment:
We might want to save `dt` for the differential in the `t` direction like `dx, dy, dz`. What do you think about `ddt` or `partial_t`?

Reply:
I haven't seen enough `d/dt`s in practice being used that way. With the major exception being `t` as a parameter of a curve. Excluding capital `T` as well.

Saying that though, I am sure there are some major exceptions I haven’t thought of where this might occur. Maybe in some GCM?

I will keep an eye open where there is a differential equation in time, defined along a local parameterization `t`. Surely there is some 1D model that does this, and I will see what notation they use.

Reply:
I haven't been able to think up any examples, so I am going to put a pin in this and merge now.